packages/aquila-stylelint/index.js

```diff
@@ -10,6 +10,8 @@ module.exports = {
     "rules": {
         "@stylistic/declaration-block-semicolon-newline-after": "always",
         "@stylistic/indentation": 4,
+        "@stylistic/no-empty-first-line": null,
```

Comment:
Why would we want an empty first line?

Reply:
@hisuwh Yeah we don't really want it. It's true by default so I've set it to null to ignore.

Reply:
I would say we don't want an empty first line surely?

Reply:
Was this just causing loads of errors though?

Reply:
@hisuwh Yeah it was complaining that there were empty first lines.

I have added an empty first line in the test to make sure that doesn't error.

Reply:
We can enforce it though, should be simple with `--fix`